plugins/woocommerce/tests/php/src/Internal/DataStores/Orders/OrdersTableDataStoreTests.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+class OrdersTableDataStoreTests extends WC_Unit_Test_Case {
+
+	public function test_read_from_migrated_order() {
```

Comment:
No assertions in this test (I assume that's temporary, but flagging for completeness 🙃 ).

Reply:
Yeah, seems like I added this file by accident, it's supposed to be part of #32701 . Removed in https://github.com/woocommerce/woocommerce/pull/32700/commits/fd0d3e05ddb3763742e0a3d58e20a42c0707e482